addons/hr_recruitment/models/hr_job.py

```diff
@@ -106,7 +131,12 @@ def get_alias_values(self):
     @api.model
     def create(self, vals):
         vals['favorite_user_ids'] = vals.get('favorite_user_ids', []) + [(4, self.env.uid)]
-        return super(Job, self).create(vals)
+        res = super(Job, self).create(vals)
+        new_source = self.env['hr.recruitment.source'].create({
+            'source_id': 6,
+            'job_id': res.id
+        })
```

Comment:
What is this code for?
We can add default source in views but we should not force the source for every created jobs.